tests/test_otel.py

```diff
@@ -32,6 +34,34 @@ def setup_tracing():
     return tracer, memory_exporter
 
 
+def test_no_span():
+    # With telemetry disabled, those calls should not raise
+    otel = OpenTelemetry(enabled=False)
+    with otel.span(
+        "GET",
+        endpoint_id="ml.open_job",
+        path_parts={"job_id": "my-job"},
+    ) as span:
+        span.set_db_statement(JsonSerializer().dumps({"timeout": "1m"}))
+        span.set_node_metadata(
+            "localhost",
+            9200,
+            "http://localhost:9200/",
+            "_ml/anomaly_detectors/my-job/_open",
+        )
+        span.set_elastic_cloud_metadata(
+            {
+                "X-Found-Handling-Cluster": "e9106fc68e3044f0b1475b04bf4ffd5f",
+                "X-Found-Handling-Instance": "instance-0000000001",
+            }
+        )
+
+
+def test_enabled():
+    otel = OpenTelemetry()
+    assert otel.enabled == bool(os.environ.get(ENABLED_ENV_VAR, "false") != "false")
```

Comment:
I don't think you need the bool

Reply:
Without it, I get the following error:


```
def test_enabled():
        otel = OpenTelemetry()
>       assert otel.enabled == os.environ.get(ENABLED_ENV_VAR, "false") != "false"
E       AssertionError: assert False == 'false'
E        +  where False = <elastic_transport._otel.OpenTelemetry object at 0x7ff727c85ed0>.enabled
E        +  and   'false' = <bound method Mapping.get of environ({'SELENIUM_JAR_PATH': '/usr/share/java/selenium-server.jar', 'CONDA': '/usr/share/miniconda', 'GITHUB_WORKSPACE': '/home/runner/work/elastic-transport-python/elastic-transport-python', 'JAVA_HOME_11_X64': '/usr/lib/jvm/temurin-11-jdk-amd64', 'PYTHON_VERSION': '3.7', 'GITHUB_PATH': '/home/runner/work/_temp/_runner_file_commands/add_path_dd17c4e8-3b90-4386-87f6-13c2712b91d4', 'GITHUB_ACTION': '__run_2', 'JAVA_HOME': '/usr/lib/jvm/temurin-11-jdk-amd64', 'GITHUB_RUN_NUMBER': '562', 'RUNNER_NAME': 'GitHub Actions 5', 'GRADLE_HOME': '/usr/share/gradle-8.6', 'GITHUB_REPOSITORY_OWNER_ID': '6764390', 'ACTIONS_RUNNER_ACTION_ARCHIVE_CACHE': '/opt/actionarchivecache', 'XDG_CONFIG_HOME': '/home/runner/.config', 'DOTNET_SKIP_FIRST_TIME_EXPERIENCE': '1', 'ANT_HOME': '/usr/share/ant', 'JAVA_HOME_8_X64': '/usr/lib/jvm/temurin-8-jdk-amd64', 'GITHUB_TRIGGERING_ACTOR': 'pquentin', 'pythonLocation': '/opt/hostedtoolcache/Python/3.12.2/x64', 'GITHUB_REF_TYPE': 'branch', 'HOMEBREW_CLEANUP_PERIODIC_FULL_DAYS': '3650', 'ANDROID_NDK': '/usr/local/lib/android/sdk/ndk/25.2.9519653', 'BOOTSTRAP_HASKELL_NONINTERACTIVE': '1', 'PWD': '/home/runner/work/elastic-transport-python/elastic-transport-python', 'PIPX_BIN_DIR': '/opt/pipx_bin', 'STATS_TRP': 'true', 'GOROOT_1_20_X64': '/opt/hostedtoolcache/go/1.20.14/x64', 'GITHUB_REPOSITORY_ID': '298402926', 'DEPLOYMENT_BASEPATH': '/opt/runner', 'GITHUB_ACTIONS': 'true', 'STATS_VMD': 'true', 'ANDROID_NDK_LATEST_HOME': '/usr/local/lib/android/sdk/ndk/26.2.11394342', 'SYSTEMD_EXEC_PID': '600', 'GITHUB_SHA': '801d86cc62e476db497bc100787ce5831d639b3d', 'GITHUB_WORKFLOW_REF': 'elastic/elastic-transport-python/.github/workflows/ci.yml@refs/pull/155/merge', 'POWERSHELL_DISTRIBUTION_CHANNEL': 'GitHub-Actions-ubuntu22', 'RUNNER_ENVIRONMENT': 'github-hosted', 'STATS_EXTP': 'https://provjobdsettingscdn.blob.core.windows.net/settings/provjobdsettings-0.5.154/provjobd.data', 'AIOHTTP_NO_EXTENSIONS': '1', 'DOTNET_MULTILEVEL_LOOKUP': '0', 'STATS_TIS': 'mining', 'GITHUB_REF': 'refs/pull/155/merge', 'RUNNER_OS': 'Linux', 'GITHUB_REF_PROTECTED': 'false', 'HOME': '/home/runner', 'GITHUB_API_URL': 'https://api.github.com/', 'LANG': 'C.UTF-8', 'RUNNER_TRACKING_ID': 'github_53226e32-4c62-4e1a-8def-7f7c2f7818fc', 'RUNNER_ARCH': 'X64', 'GOROOT_1_21_X64': '/opt/hostedtoolcache/go/1.21.8/x64', 'RUNNER_TEMP': '/home/runner/work/_temp', 'GITHUB_STATE': '/home/runner/work/_temp/_runner_file_commands/save_state_dd17c4e8-3b90-4386-87f6-13c2712b91d4', 'EDGEWEBDRIVER': '/usr/local/share/edge_driver', 'JAVA_HOME_21_X64': '/usr/lib/jvm/temurin-21-jdk-amd64', 'GITHUB_ENV': '/home/runner/work/_temp/_runner_file_commands/set_env_dd17c4e8-3b90-4386-87f6-13c2712b91d4', 'GITHUB_EVENT_PATH': '/home/runner/work/_temp/_github_workflow/event.json', 'INVOCATION_ID': 'c3a78028f5bc473bba74241751a97edb', 'GITHUB_EVENT_NAME': 'pull_request', 'GITHUB_RUN_ID': '8415653306', 'JAVA_HOME_17_X64': '/usr/lib/jvm/temurin-17-jdk-amd64', 'ANDROID_NDK_HOME': '/usr/local/lib/android/sdk/ndk/25.2.9519653', 'GITHUB_STEP_SUMMARY': '/home/runner/work/_temp/_runner_file_commands/step_summary_dd17c4e8-3b90-4386-87f6-13c2712b91d4', 'HOMEBREW_NO_AUTO_UPDATE': '1', 'GITHUB_ACTOR': 'pquentin', 'NVM_DIR': '/home/runner/.nvm', 'SGX_AESM_ADDR': '1', 'GITHUB_RUN_ATTEMPT': '1', 'STATS_RDCL': 'true', 'ANDROID_HOME': '/usr/local/lib/android/sdk', 'GITHUB_GRAPHQL_URL': 'https://api.github.com/graphql', 'ACCEPT_EULA': 'Y', 'RUNNER_USER': 'runner', 'STATS_UE': 'true', 'USER': 'runner', 'GITHUB_SERVER_URL': 'https://github.com/', 'STATS_V3PS': 'true', 'PIPX_HOME': '/opt/pipx', 'GECKOWEBDRIVER': '/usr/local/share/gecko_driver', 'STATS_EXT': 'true', 'CHROMEWEBDRIVER': '/usr/local/share/chromedriver-linux64', 'SHLVL': '1', 'YARL_NO_EXTENSIONS': '1', 'ANDROID_SDK_ROOT': '/usr/local/lib/android/sdk', 'VCPKG_INSTALLATION_ROOT': '/usr/local/share/vcpkg', 'GITHUB_ACTOR_ID': '42327', 'RUNNER_TOOL_CACHE': '/opt/hostedtoolcache', 'ImageVersion': '20240317.1.0', 'DOTNET_NOLOGO': '1', 'GITHUB_WORKFLOW_SHA': '801d86cc62e476db497bc100787ce5831d639b3d', 'GITHUB_REF_NAME': '155/merge', 'GITHUB_JOB': 'test', 'LD_LIBRARY_PATH': '/opt/hostedtoolcache/Python/3.12.2/x64/lib:/opt/hostedtoolcache/Python/3.7.17/x64/lib', 'XDG_RUNTIME_DIR': '/run/user/1001', 'AZURE_EXTENSION_DIR': '/opt/az/azcliextensions', 'PERFLOG_LOCATION_SETTING': 'RUNNER_PERFLOG', 'GITHUB_REPOSITORY': 'elastic/elastic-transport-python', 'ANDROID_NDK_ROOT': '/usr/local/lib/android/sdk/ndk/25.2.9519653', 'GOROOT_1_22_X64': '/opt/hostedtoolcache/go/1.22.1/x64', 'CHROME_BIN': '/usr/bin/google-chrome', 'GITHUB_RETENTION_DAYS': '90', 'JOURNAL_STREAM': '8:18886', 'RUNNER_WORKSPACE': '/home/runner/work/elastic-transport-python', 'LEIN_HOME': '/usr/local/lib/lein', 'LEIN_JAR': '/usr/local/lib/lein/self-installs/leiningen-2.11.2-standalone.jar', 'GITHUB_ACTION_REPOSITORY': '', 'PATH': '/home/runner/work/elastic-transport-python/elastic-transport-python/.nox/test-3-7/bin:/opt/hostedtoolcache/Python/3.12.2/x64/bin:/opt/hostedtoolcache/Python/3.12.2/x64:/opt/hostedtoolcache/Python/3.7.17/x64/bin:/opt/hostedtoolcache/Python/3.7.17/x64:/snap/bin:/home/runner/.local/bin:/opt/pipx_bin:/home/runner/.cargo/bin:/home/runner/.config/composer/vendor/bin:/usr/local/.ghcup/bin:/home/runner/.dotnet/tools:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/games:/usr/local/games:/snap/bin', 'RUNNER_PERFLOG': '/home/runner/perflog', 'GITHUB_BASE_REF': 'main', 'GHCUP_INSTALL_BASE_PREFIX': '/usr/local', 'CI': 'true', 'SWIFT_PATH': '/usr/share/swift/usr/bin', 'ImageOS': 'ubuntu22', 'GITHUB_REPOSITORY_OWNER': 'elastic', 'GITHUB_HEAD_REF': 'otel-db.statement', 'GITHUB_ACTION_REF': '', 'GITHUB_WORKFLOW': 'CI', 'DEBIAN_FRONTEND': 'noninteractive', 'GITHUB_OUTPUT': '/home/runner/work/_temp/_runner_file_commands/set_output_dd17c4e8-3b90-4386-87f6-13c2712b91d4', 'AGENT_TOOLSDIRECTORY': '/opt/hostedtoolcache', 'FROZENLIST_NO_EXTENSIONS': '1', '_': '/opt/hostedtoolcache/Python/3.12.2/x64/bin/nox', 'VIRTUAL_ENV': '/home/runner/work/elastic-transport-python/elastic-transport-python/.nox/test-3-7', 'NOX_CURRENT_SESSION': 'test-3.7', 'PYTHONWARNINGS': 'always::DeprecationWarning', 'COV_CORE_SOURCE': 'elastic_transport:elastic_transport', 'COV_CORE_CONFIG': ':', 'COV_CORE_DATAFILE': '/home/runner/work/elastic-transport-python/elastic-transport-python/.coverage', 'PYTEST_CURRENT_TEST': 'tests/test_otel.py::test_enabled (call)'})>('OTEL_PYTHON_INSTRUMENTATION_ELASTICSEARCH_ENABLED', 'false')
E        +    where <bound method Mapping.get of environ({'SELENIUM_JAR_PATH': '/usr/share/java/selenium-server.jar', 'CONDA': '/usr/share/miniconda', 'GITHUB_WORKSPACE': '/home/runner/work/elastic-transport-python/elastic-transport-python', 'JAVA_HOME_11_X64': '/usr/lib/jvm/temurin-11-jdk-amd64', 'PYTHON_VERSION': '3.7', 'GITHUB_PATH': '/home/runner/work/_temp/_runner_file_commands/add_path_dd17c4e8-3b90-4386-87f6-13c2712b91d4', 'GITHUB_ACTION': '__run_2', 'JAVA_HOME': '/usr/lib/jvm/temurin-11-jdk-amd64', 'GITHUB_RUN_NUMBER': '562', 'RUNNER_NAME': 'GitHub Actions 5', 'GRADLE_HOME': '/usr/share/gradle-8.6', 'GITHUB_REPOSITORY_OWNER_ID': '6764390', 'ACTIONS_RUNNER_ACTION_ARCHIVE_CACHE': '/opt/actionarchivecache', 'XDG_CONFIG_HOME': '/home/runner/.config', 'DOTNET_SKIP_FIRST_TIME_EXPERIENCE': '1', 'ANT_HOME': '/usr/share/ant', 'JAVA_HOME_8_X64': '/usr/lib/jvm/temurin-8-jdk-amd64', 'GITHUB_TRIGGERING_ACTOR': 'pquentin', 'pythonLocation': '/opt/hostedtoolcache/Python/3.12.2/x64', 'GITHUB_REF_TYPE': 'branch', 'HOMEBREW_CLEANUP_PERIODIC_FULL_DAYS': '3650', 'ANDROID_NDK': '/usr/local/lib/android/sdk/ndk/25.2.9519653', 'BOOTSTRAP_HASKELL_NONINTERACTIVE': '1', 'PWD': '/home/runner/work/elastic-transport-python/elastic-transport-python', 'PIPX_BIN_DIR': '/opt/pipx_bin', 'STATS_TRP': 'true', 'GOROOT_1_20_X64': '/opt/hostedtoolcache/go/1.20.14/x64', 'GITHUB_REPOSITORY_ID': '298402926', 'DEPLOYMENT_BASEPATH': '/opt/runner', 'GITHUB_ACTIONS': 'true', 'STATS_VMD': 'true', 'ANDROID_NDK_LATEST_HOME': '/usr/local/lib/android/sdk/ndk/26.2.11394342', 'SYSTEMD_EXEC_PID': '600', 'GITHUB_SHA': '801d86cc62e476db497bc100787ce5831d639b3d', 'GITHUB_WORKFLOW_REF': 'elastic/elastic-transport-python/.github/workflows/ci.yml@refs/pull/155/merge', 'POWERSHELL_DISTRIBUTION_CHANNEL': 'GitHub-Actions-ubuntu22', 'RUNNER_ENVIRONMENT': 'github-hosted', 'STATS_EXTP': 'https://provjobdsettingscdn.blob.core.windows.net/settings/provjobdsettings-0.5.154/provjobd.data', 'AIOHTTP_NO_EXTENSIONS': '1', 'DOTNET_MULTILEVEL_LOOKUP': '0', 'STATS_TIS': 'mining', 'GITHUB_REF': 'refs/pull/155/merge', 'RUNNER_OS': 'Linux', 'GITHUB_REF_PROTECTED': 'false', 'HOME': '/home/runner', 'GITHUB_API_URL': 'https://api.github.com/', 'LANG': 'C.UTF-8', 'RUNNER_TRACKING_ID': 'github_53226e32-4c62-4e1a-8def-7f7c2f7818fc', 'RUNNER_ARCH': 'X64', 'GOROOT_1_21_X64': '/opt/hostedtoolcache/go/1.21.8/x64', 'RUNNER_TEMP': '/home/runner/work/_temp', 'GITHUB_STATE': '/home/runner/work/_temp/_runner_file_commands/save_state_dd17c4e8-3b90-4386-87f6-13c2712b91d4', 'EDGEWEBDRIVER': '/usr/local/share/edge_driver', 'JAVA_HOME_21_X64': '/usr/lib/jvm/temurin-21-jdk-amd64', 'GITHUB_ENV': '/home/runner/work/_temp/_runner_file_commands/set_env_dd17c4e8-3b90-4386-87f6-13c2712b91d4', 'GITHUB_EVENT_PATH': '/home/runner/work/_temp/_github_workflow/event.json', 'INVOCATION_ID': 'c3a78028f5bc473bba74241751a97edb', 'GITHUB_EVENT_NAME': 'pull_request', 'GITHUB_RUN_ID': '8415653306', 'JAVA_HOME_17_X64': '/usr/lib/jvm/temurin-17-jdk-amd64', 'ANDROID_NDK_HOME': '/usr/local/lib/android/sdk/ndk/25.2.9519653', 'GITHUB_STEP_SUMMARY': '/home/runner/work/_temp/_runner_file_commands/step_summary_dd17c4e8-3b90-4386-87f6-13c2712b91d4', 'HOMEBREW_NO_AUTO_UPDATE': '1', 'GITHUB_ACTOR': 'pquentin', 'NVM_DIR': '/home/runner/.nvm', 'SGX_AESM_ADDR': '1', 'GITHUB_RUN_ATTEMPT': '1', 'STATS_RDCL': 'true', 'ANDROID_HOME': '/usr/local/lib/android/sdk', 'GITHUB_GRAPHQL_URL': 'https://api.github.com/graphql', 'ACCEPT_EULA': 'Y', 'RUNNER_USER': 'runner', 'STATS_UE': 'true', 'USER': 'runner', 'GITHUB_SERVER_URL': 'https://github.com/', 'STATS_V3PS': 'true', 'PIPX_HOME': '/opt/pipx', 'GECKOWEBDRIVER': '/usr/local/share/gecko_driver', 'STATS_EXT': 'true', 'CHROMEWEBDRIVER': '/usr/local/share/chromedriver-linux64', 'SHLVL': '1', 'YARL_NO_EXTENSIONS': '1', 'ANDROID_SDK_ROOT': '/usr/local/lib/android/sdk', 'VCPKG_INSTALLATION_ROOT': '/usr/local/share/vcpkg', 'GITHUB_ACTOR_ID': '42327', 'RUNNER_TOOL_CACHE': '/opt/hostedtoolcache', 'ImageVersion': '20240317.1.0', 'DOTNET_NOLOGO': '1', 'GITHUB_WORKFLOW_SHA': '801d86cc62e476db497bc100787ce5831d639b3d', 'GITHUB_REF_NAME': '155/merge', 'GITHUB_JOB': 'test', 'LD_LIBRARY_PATH': '/opt/hostedtoolcache/Python/3.12.2/x64/lib:/opt/hostedtoolcache/Python/3.7.17/x64/lib', 'XDG_RUNTIME_DIR': '/run/user/1001', 'AZURE_EXTENSION_DIR': '/opt/az/azcliextensions', 'PERFLOG_LOCATION_SETTING': 'RUNNER_PERFLOG', 'GITHUB_REPOSITORY': 'elastic/elastic-transport-python', 'ANDROID_NDK_ROOT': '/usr/local/lib/android/sdk/ndk/25.2.9519653', 'GOROOT_1_22_X64': '/opt/hostedtoolcache/go/1.22.1/x64', 'CHROME_BIN': '/usr/bin/google-chrome', 'GITHUB_RETENTION_DAYS': '90', 'JOURNAL_STREAM': '8:18886', 'RUNNER_WORKSPACE': '/home/runner/work/elastic-transport-python', 'LEIN_HOME': '/usr/local/lib/lein', 'LEIN_JAR': '/usr/local/lib/lein/self-installs/leiningen-2.11.2-standalone.jar', 'GITHUB_ACTION_REPOSITORY': '', 'PATH': '/home/runner/work/elastic-transport-python/elastic-transport-python/.nox/test-3-7/bin:/opt/hostedtoolcache/Python/3.12.2/x64/bin:/opt/hostedtoolcache/Python/3.12.2/x64:/opt/hostedtoolcache/Python/3.7.17/x64/bin:/opt/hostedtoolcache/Python/3.7.17/x64:/snap/bin:/home/runner/.local/bin:/opt/pipx_bin:/home/runner/.cargo/bin:/home/runner/.config/composer/vendor/bin:/usr/local/.ghcup/bin:/home/runner/.dotnet/tools:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/games:/usr/local/games:/snap/bin', 'RUNNER_PERFLOG': '/home/runner/perflog', 'GITHUB_BASE_REF': 'main', 'GHCUP_INSTALL_BASE_PREFIX': '/usr/local', 'CI': 'true', 'SWIFT_PATH': '/usr/share/swift/usr/bin', 'ImageOS': 'ubuntu22', 'GITHUB_REPOSITORY_OWNER': 'elastic', 'GITHUB_HEAD_REF': 'otel-db.statement', 'GITHUB_ACTION_REF': '', 'GITHUB_WORKFLOW': 'CI', 'DEBIAN_FRONTEND': 'noninteractive', 'GITHUB_OUTPUT': '/home/runner/work/_temp/_runner_file_commands/set_output_dd17c4e8-3b90-4386-87f6-13c2712b91d4', 'AGENT_TOOLSDIRECTORY': '/opt/hostedtoolcache', 'FROZENLIST_NO_EXTENSIONS': '1', '_': '/opt/hostedtoolcache/Python/3.12.2/x64/bin/nox', 'VIRTUAL_ENV': '/home/runner/work/elastic-transport-python/elastic-transport-python/.nox/test-3-7', 'NOX_CURRENT_SESSION': 'test-3.7', 'PYTHONWARNINGS': 'always::DeprecationWarning', 'COV_CORE_SOURCE': 'elastic_transport:elastic_transport', 'COV_CORE_CONFIG': ':', 'COV_CORE_DATAFILE': '/home/runner/work/elastic-transport-python/elastic-transport-python/.coverage', 'PYTEST_CURRENT_TEST': 'tests/test_otel.py::test_enabled (call)'})> = environ({'SELENIUM_JAR_PATH': '/usr/share/java/selenium-server.jar', 'CONDA': '/usr/share/miniconda', 'GITHUB_WORKSPACE': '/home/runner/work/elastic-transport-python/elastic-transport-python', 'JAVA_HOME_11_X64': '/usr/lib/jvm/temurin-11-jdk-amd64', 'PYTHON_VERSION': '3.7', 'GITHUB_PATH': '/home/runner/work/_temp/_runner_file_commands/add_path_dd17c4e8-3b90-4386-87f6-13c2712b91d4', 'GITHUB_ACTION': '__run_2', 'JAVA_HOME': '/usr/lib/jvm/temurin-11-jdk-amd64', 'GITHUB_RUN_NUMBER': '562', 'RUNNER_NAME': 'GitHub Actions 5', 'GRADLE_HOME': '/usr/share/gradle-8.6', 'GITHUB_REPOSITORY_OWNER_ID': '6764390', 'ACTIONS_RUNNER_ACTION_ARCHIVE_CACHE': '/opt/actionarchivecache', 'XDG_CONFIG_HOME': '/home/runner/.config', 'DOTNET_SKIP_FIRST_TIME_EXPERIENCE': '1', 'ANT_HOME': '/usr/share/ant', 'JAVA_HOME_8_X64': '/usr/lib/jvm/temurin-8-jdk-amd64', 'GITHUB_TRIGGERING_ACTOR': 'pquentin', 'pythonLocation': '/opt/hostedtoolcache/Python/3.12.2/x64', 'GITHUB_REF_TYPE': 'branch', 'HOMEBREW_CLEANUP_PERIODIC_FULL_DAYS': '3650', 'ANDROID_NDK': '/usr/local/lib/android/sdk/ndk/25.2.9519653', 'BOOTSTRAP_HASKELL_NONINTERACTIVE': '1', 'PWD': '/home/runner/work/elastic-transport-python/elastic-transport-python', 'PIPX_BIN_DIR': '/opt/pipx_bin', 'STATS_TRP': 'true', 'GOROOT_1_20_X64': '/opt/hostedtoolcache/go/1.20.14/x64', 'GITHUB_REPOSITORY_ID': '298402926', 'DEPLOYMENT_BASEPATH': '/opt/runner', 'GITHUB_ACTIONS': 'true', 'STATS_VMD': 'true', 'ANDROID_NDK_LATEST_HOME': '/usr/local/lib/android/sdk/ndk/26.2.11394342', 'SYSTEMD_EXEC_PID': '600', 'GITHUB_SHA': '801d86cc62e476db497bc100787ce5831d639b3d', 'GITHUB_WORKFLOW_REF': 'elastic/elastic-transport-python/.github/workflows/ci.yml@refs/pull/155/merge', 'POWERSHELL_DISTRIBUTION_CHANNEL': 'GitHub-Actions-ubuntu22', 'RUNNER_ENVIRONMENT': 'github-hosted', 'STATS_EXTP': 'https://provjobdsettingscdn.blob.core.windows.net/settings/provjobdsettings-0.5.154/provjobd.data', 'AIOHTTP_NO_EXTENSIONS': '1', 'DOTNET_MULTILEVEL_LOOKUP': '0', 'STATS_TIS': 'mining', 'GITHUB_REF': 'refs/pull/155/merge', 'RUNNER_OS': 'Linux', 'GITHUB_REF_PROTECTED': 'false', 'HOME': '/home/runner', 'GITHUB_API_URL': 'https://api.github.com/', 'LANG': 'C.UTF-8', 'RUNNER_TRACKING_ID': 'github_53226e32-4c62-4e1a-8def-7f7c2f7818fc', 'RUNNER_ARCH': 'X64', 'GOROOT_1_21_X64': '/opt/hostedtoolcache/go/1.21.8/x64', 'RUNNER_TEMP': '/home/runner/work/_temp', 'GITHUB_STATE': '/home/runner/work/_temp/_runner_file_commands/save_state_dd17c4e8-3b90-4386-87f6-13c2712b91d4', 'EDGEWEBDRIVER': '/usr/local/share/edge_driver', 'JAVA_HOME_21_X64': '/usr/lib/jvm/temurin-21-jdk-amd64', 'GITHUB_ENV': '/home/runner/work/_temp/_runner_file_commands/set_env_dd17c4e8-3b90-4386-87f6-13c2712b91d4', 'GITHUB_EVENT_PATH': '/home/runner/work/_temp/_github_workflow/event.json', 'INVOCATION_ID': 'c3a78028f5bc473bba74241751a97edb', 'GITHUB_EVENT_NAME': 'pull_request', 'GITHUB_RUN_ID': '8415653306', 'JAVA_HOME_17_X64': '/usr/lib/jvm/temurin-17-jdk-amd64', 'ANDROID_NDK_HOME': '/usr/local/lib/android/sdk/ndk/25.2.9519653', 'GITHUB_STEP_SUMMARY': '/home/runner/work/_temp/_runner_file_commands/step_summary_dd17c4e8-3b90-4386-87f6-13c2712b91d4', 'HOMEBREW_NO_AUTO_UPDATE': '1', 'GITHUB_ACTOR': 'pquentin', 'NVM_DIR': '/home/runner/.nvm', 'SGX_AESM_ADDR': '1', 'GITHUB_RUN_ATTEMPT': '1', 'STATS_RDCL': 'true', 'ANDROID_HOME': '/usr/local/lib/android/sdk', 'GITHUB_GRAPHQL_URL': 'https://api.github.com/graphql', 'ACCEPT_EULA': 'Y', 'RUNNER_USER': 'runner', 'STATS_UE': 'true', 'USER': 'runner', 'GITHUB_SERVER_URL': 'https://github.com/', 'STATS_V3PS': 'true', 'PIPX_HOME': '/opt/pipx', 'GECKOWEBDRIVER': '/usr/local/share/gecko_driver', 'STATS_EXT': 'true', 'CHROMEWEBDRIVER': '/usr/local/share/chromedriver-linux64', 'SHLVL': '1', 'YARL_NO_EXTENSIONS': '1', 'ANDROID_SDK_ROOT': '/usr/local/lib/android/sdk', 'VCPKG_INSTALLATION_ROOT': '/usr/local/share/vcpkg', 'GITHUB_ACTOR_ID': '42327', 'RUNNER_TOOL_CACHE': '/opt/hostedtoolcache', 'ImageVersion': '20240317.1.0', 'DOTNET_NOLOGO': '1', 'GITHUB_WORKFLOW_SHA': '801d86cc62e476db497bc100787ce5831d639b3d', 'GITHUB_REF_NAME': '155/merge', 'GITHUB_JOB': 'test', 'LD_LIBRARY_PATH': '/opt/hostedtoolcache/Python/3.12.2/x64/lib:/opt/hostedtoolcache/Python/3.7.17/x64/lib', 'XDG_RUNTIME_DIR': '/run/user/1001', 'AZURE_EXTENSION_DIR': '/opt/az/azcliextensions', 'PERFLOG_LOCATION_SETTING': 'RUNNER_PERFLOG', 'GITHUB_REPOSITORY': 'elastic/elastic-transport-python', 'ANDROID_NDK_ROOT': '/usr/local/lib/android/sdk/ndk/25.2.9519653', 'GOROOT_1_22_X64': '/opt/hostedtoolcache/go/1.22.1/x64', 'CHROME_BIN': '/usr/bin/google-chrome', 'GITHUB_RETENTION_DAYS': '90', 'JOURNAL_STREAM': '8:18886', 'RUNNER_WORKSPACE': '/home/runner/work/elastic-transport-python', 'LEIN_HOME': '/usr/local/lib/lein', 'LEIN_JAR': '/usr/local/lib/lein/self-installs/leiningen-2.11.2-standalone.jar', 'GITHUB_ACTION_REPOSITORY': '', 'PATH': '/home/runner/work/elastic-transport-python/elastic-transport-python/.nox/test-3-7/bin:/opt/hostedtoolcache/Python/3.12.2/x64/bin:/opt/hostedtoolcache/Python/3.12.2/x64:/opt/hostedtoolcache/Python/3.7.17/x64/bin:/opt/hostedtoolcache/Python/3.7.17/x64:/snap/bin:/home/runner/.local/bin:/opt/pipx_bin:/home/runner/.cargo/bin:/home/runner/.config/composer/vendor/bin:/usr/local/.ghcup/bin:/home/runner/.dotnet/tools:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/games:/usr/local/games:/snap/bin', 'RUNNER_PERFLOG': '/home/runner/perflog', 'GITHUB_BASE_REF': 'main', 'GHCUP_INSTALL_BASE_PREFIX': '/usr/local', 'CI': 'true', 'SWIFT_PATH': '/usr/share/swift/usr/bin', 'ImageOS': 'ubuntu22', 'GITHUB_REPOSITORY_OWNER': 'elastic', 'GITHUB_HEAD_REF': 'otel-db.statement', 'GITHUB_ACTION_REF': '', 'GITHUB_WORKFLOW': 'CI', 'DEBIAN_FRONTEND': 'noninteractive', 'GITHUB_OUTPUT': '/home/runner/work/_temp/_runner_file_commands/set_output_dd17c4e8-3b90-4386-87f6-13c2712b91d4', 'AGENT_TOOLSDIRECTORY': '/opt/hostedtoolcache', 'FROZENLIST_NO_EXTENSIONS': '1', '_': '/opt/hostedtoolcache/Python/3.12.2/x64/bin/nox', 'VIRTUAL_ENV': '/home/runner/work/elastic-transport-python/elastic-transport-python/.nox/test-3-7', 'NOX_CURRENT_SESSION': 'test-3.7', 'PYTHONWARNINGS': 'always::DeprecationWarning', 'COV_CORE_SOURCE': 'elastic_transport:elastic_transport', 'COV_CORE_CONFIG': ':', 'COV_CORE_DATAFILE': '/home/runner/work/elastic-transport-python/elastic-transport-python/.coverage', 'PYTEST_CURRENT_TEST': 'tests/test_otel.py::test_enabled (call)'}).get
E        +      where environ({'SELENIUM_JAR_PATH': '/usr/share/java/selenium-server.jar', 'CONDA': '/usr/share/miniconda', 'GITHUB_WORKSPACE': '/home/runner/work/elastic-transport-python/elastic-transport-python', 'JAVA_HOME_11_X64': '/usr/lib/jvm/temurin-11-jdk-amd64', 'PYTHON_VERSION': '3.7', 'GITHUB_PATH': '/home/runner/work/_temp/_runner_file_commands/add_path_dd17c4e8-3b90-4386-87f6-13c2712b91d4', 'GITHUB_ACTION': '__run_2', 'JAVA_HOME': '/usr/lib/jvm/temurin-11-jdk-amd64', 'GITHUB_RUN_NUMBER': '562', 'RUNNER_NAME': 'GitHub Actions 5', 'GRADLE_HOME': '/usr/share/gradle-8.6', 'GITHUB_REPOSITORY_OWNER_ID': '6764390', 'ACTIONS_RUNNER_ACTION_ARCHIVE_CACHE': '/opt/actionarchivecache', 'XDG_CONFIG_HOME': '/home/runner/.config', 'DOTNET_SKIP_FIRST_TIME_EXPERIENCE': '1', 'ANT_HOME': '/usr/share/ant', 'JAVA_HOME_8_X64': '/usr/lib/jvm/temurin-8-jdk-amd64', 'GITHUB_TRIGGERING_ACTOR': 'pquentin', 'pythonLocation': '/opt/hostedtoolcache/Python/3.12.2/x64', 'GITHUB_REF_TYPE': 'branch', 'HOMEBREW_CLEANUP_PERIODIC_FULL_DAYS': '3650', 'ANDROID_NDK': '/usr/local/lib/android/sdk/ndk/25.2.9519653', 'BOOTSTRAP_HASKELL_NONINTERACTIVE': '1', 'PWD': '/home/runner/work/elastic-transport-python/elastic-transport-python', 'PIPX_BIN_DIR': '/opt/pipx_bin', 'STATS_TRP': 'true', 'GOROOT_1_20_X64': '/opt/hostedtoolcache/go/1.20.14/x64', 'GITHUB_REPOSITORY_ID': '298402926', 'DEPLOYMENT_BASEPATH': '/opt/runner', 'GITHUB_ACTIONS': 'true', 'STATS_VMD': 'true', 'ANDROID_NDK_LATEST_HOME': '/usr/local/lib/android/sdk/ndk/26.2.11394342', 'SYSTEMD_EXEC_PID': '600', 'GITHUB_SHA': '801d86cc62e476db497bc100787ce5831d639b3d', 'GITHUB_WORKFLOW_REF': 'elastic/elastic-transport-python/.github/workflows/ci.yml@refs/pull/155/merge', 'POWERSHELL_DISTRIBUTION_CHANNEL': 'GitHub-Actions-ubuntu22', 'RUNNER_ENVIRONMENT': 'github-hosted', 'STATS_EXTP': 'https://provjobdsettingscdn.blob.core.windows.net/settings/provjobdsettings-0.5.154/provjobd.data', 'AIOHTTP_NO_EXTENSIONS': '1', 'DOTNET_MULTILEVEL_LOOKUP': '0', 'STATS_TIS': 'mining', 'GITHUB_REF': 'refs/pull/155/merge', 'RUNNER_OS': 'Linux', 'GITHUB_REF_PROTECTED': 'false', 'HOME': '/home/runner', 'GITHUB_API_URL': 'https://api.github.com/', 'LANG': 'C.UTF-8', 'RUNNER_TRACKING_ID': 'github_53226e32-4c62-4e1a-8def-7f7c2f7818fc', 'RUNNER_ARCH': 'X64', 'GOROOT_1_21_X64': '/opt/hostedtoolcache/go/1.21.8/x64', 'RUNNER_TEMP': '/home/runner/work/_temp', 'GITHUB_STATE': '/home/runner/work/_temp/_runner_file_commands/save_state_dd17c4e8-3b90-4386-87f6-13c2712b91d4', 'EDGEWEBDRIVER': '/usr/local/share/edge_driver', 'JAVA_HOME_21_X64': '/usr/lib/jvm/temurin-21-jdk-amd64', 'GITHUB_ENV': '/home/runner/work/_temp/_runner_file_commands/set_env_dd17c4e8-3b90-4386-87f6-13c2712b91d4', 'GITHUB_EVENT_PATH': '/home/runner/work/_temp/_github_workflow/event.json', 'INVOCATION_ID': 'c3a78028f5bc473bba74241751a97edb', 'GITHUB_EVENT_NAME': 'pull_request', 'GITHUB_RUN_ID': '8415653306', 'JAVA_HOME_17_X64': '/usr/lib/jvm/temurin-17-jdk-amd64', 'ANDROID_NDK_HOME': '/usr/local/lib/android/sdk/ndk/25.2.9519653', 'GITHUB_STEP_SUMMARY': '/home/runner/work/_temp/_runner_file_commands/step_summary_dd17c4e8-3b90-4386-87f6-13c2712b91d4', 'HOMEBREW_NO_AUTO_UPDATE': '1', 'GITHUB_ACTOR': 'pquentin', 'NVM_DIR': '/home/runner/.nvm', 'SGX_AESM_ADDR': '1', 'GITHUB_RUN_ATTEMPT': '1', 'STATS_RDCL': 'true', 'ANDROID_HOME': '/usr/local/lib/android/sdk', 'GITHUB_GRAPHQL_URL': 'https://api.github.com/graphql', 'ACCEPT_EULA': 'Y', 'RUNNER_USER': 'runner', 'STATS_UE': 'true', 'USER': 'runner', 'GITHUB_SERVER_URL': 'https://github.com/', 'STATS_V3PS': 'true', 'PIPX_HOME': '/opt/pipx', 'GECKOWEBDRIVER': '/usr/local/share/gecko_driver', 'STATS_EXT': 'true', 'CHROMEWEBDRIVER': '/usr/local/share/chromedriver-linux64', 'SHLVL': '1', 'YARL_NO_EXTENSIONS': '1', 'ANDROID_SDK_ROOT': '/usr/local/lib/android/sdk', 'VCPKG_INSTALLATION_ROOT': '/usr/local/share/vcpkg', 'GITHUB_ACTOR_ID': '42327', 'RUNNER_TOOL_CACHE': '/opt/hostedtoolcache', 'ImageVersion': '20240317.1.0', 'DOTNET_NOLOGO': '1', 'GITHUB_WORKFLOW_SHA': '801d86cc62e476db497bc100787ce5831d639b3d', 'GITHUB_REF_NAME': '155/merge', 'GITHUB_JOB': 'test', 'LD_LIBRARY_PATH': '/opt/hostedtoolcache/Python/3.12.2/x64/lib:/opt/hostedtoolcache/Python/3.7.17/x64/lib', 'XDG_RUNTIME_DIR': '/run/user/1001', 'AZURE_EXTENSION_DIR': '/opt/az/azcliextensions', 'PERFLOG_LOCATION_SETTING': 'RUNNER_PERFLOG', 'GITHUB_REPOSITORY': 'elastic/elastic-transport-python', 'ANDROID_NDK_ROOT': '/usr/local/lib/android/sdk/ndk/25.2.9519653', 'GOROOT_1_22_X64': '/opt/hostedtoolcache/go/1.22.1/x64', 'CHROME_BIN': '/usr/bin/google-chrome', 'GITHUB_RETENTION_DAYS': '90', 'JOURNAL_STREAM': '8:18886', 'RUNNER_WORKSPACE': '/home/runner/work/elastic-transport-python', 'LEIN_HOME': '/usr/local/lib/lein', 'LEIN_JAR': '/usr/local/lib/lein/self-installs/leiningen-2.11.2-standalone.jar', 'GITHUB_ACTION_REPOSITORY': '', 'PATH': '/home/runner/work/elastic-transport-python/elastic-transport-python/.nox/test-3-7/bin:/opt/hostedtoolcache/Python/3.12.2/x64/bin:/opt/hostedtoolcache/Python/3.12.2/x64:/opt/hostedtoolcache/Python/3.7.17/x64/bin:/opt/hostedtoolcache/Python/3.7.17/x64:/snap/bin:/home/runner/.local/bin:/opt/pipx_bin:/home/runner/.cargo/bin:/home/runner/.config/composer/vendor/bin:/usr/local/.ghcup/bin:/home/runner/.dotnet/tools:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/games:/usr/local/games:/snap/bin', 'RUNNER_PERFLOG': '/home/runner/perflog', 'GITHUB_BASE_REF': 'main', 'GHCUP_INSTALL_BASE_PREFIX': '/usr/local', 'CI': 'true', 'SWIFT_PATH': '/usr/share/swift/usr/bin', 'ImageOS': 'ubuntu22', 'GITHUB_REPOSITORY_OWNER': 'elastic', 'GITHUB_HEAD_REF': 'otel-db.statement', 'GITHUB_ACTION_REF': '', 'GITHUB_WORKFLOW': 'CI', 'DEBIAN_FRONTEND': 'noninteractive', 'GITHUB_OUTPUT': '/home/runner/work/_temp/_runner_file_commands/set_output_dd17c4e8-3b90-4386-87f6-13c2712b91d4', 'AGENT_TOOLSDIRECTORY': '/opt/hostedtoolcache', 'FROZENLIST_NO_EXTENSIONS': '1', '_': '/opt/hostedtoolcache/Python/3.12.2/x64/bin/nox', 'VIRTUAL_ENV': '/home/runner/work/elastic-transport-python/elastic-transport-python/.nox/test-3-7', 'NOX_CURRENT_SESSION': 'test-3.7', 'PYTHONWARNINGS': 'always::DeprecationWarning', 'COV_CORE_SOURCE': 'elastic_transport:elastic_transport', 'COV_CORE_CONFIG': ':', 'COV_CORE_DATAFILE': '/home/runner/work/elastic-transport-python/elastic-transport-python/.coverage', 'PYTEST_CURRENT_TEST': 'tests/test_otel.py::test_enabled (call)'}) = os.environ
```

But I realized I can simply use parens: `assert foo == (bar != False)`.